doc/sphinx/how-to/how-to-enable-graphics-core22-on-a-device.md

```diff
@@ -0,0 +1,455 @@
+# How to enable graphics-core22 on a device
+
+
+This document will guide you through the process of assembling a 
+`graphics-core22` snap. With this snap, your device will be ready
+to run graphical snaps like `ubuntu-frame` or any other *Mir*-based
+compositor in a snapped environment.
+
+To this end, we will accomplish the following in order:
+- How to assemble a `graphics-core22` provider snap
+- How to test a `graphics-core22` provider snap
+- How to test the confinement of your *Mir*-based snap that relies on the provider snap
+
+## Prerequisites
+Before we get started, let's make sure that we have everything that we require.
+
+### 1. Development Board
+For starters, make sure that you have some sort of development board on hand.
+
+### 2. GPU Support Check
+Next, make sure that this board runs a `snapd`-enabled Ubuntu image with
+a kernel that includes GPU support. To do this, we'll run a series of checks.
+
+First, you should ensure that *at least* `/dev/dri/card0` exists, but you 
+may have more depending on your setup. You may check this from the command line.
+
+Next, install [drm_info](https://gitlab.freedesktop.org/emersion/drm_info). The
+output of `drm_info` will look something like the following:
+```
+Node: /dev/dri/card1
+├───Driver: amdgpu (AMD GPU) version 3.54.0 (20150101)
+│   ├─── ...
+├───Device: PCI 1002:747e Advanced Micro Devices, Inc. [AMD/ATI] Device 747e
+│   └───Available nodes: primary, render
+├───Framebuffer size
+│   ├───Width: [0, 16384]
+│   └───Height: [0, 16384]
+├───Connectors
+|   |   ...
+│   ├───Connector 3
+│   ├───Object ID: 101
+│   ├───Type: HDMI-A
+│   ├───Status: connected
+│   ├───Physical size: 530x300 mm
+│   ├───Subpixel: unknown
+│   ├───Encoders: {3}
+│   ├───Modes
+│   │   ├───1920x1080@60.00 preferred driver phsync pvsync 
+│   │   ├───1920x1080@60.00 driver phsync pvsync 16:9 
+│   │   ├───...
+|   |   ...
+├───Encoders
+│   ├───Encoder 0
+│   │   ├───Object ID: 82
+│   │   ├───Type: TMDS
+│   │   ├───CRTCS: {0, 1, 2, 3}
+│   │   └───Clones: {0}
+|   |   ...
+├───CRTCs
+│   ├───CRTC 0
+│   │   ├───Object ID: 72
+│   │   ├───Legacy info
+│   │   │   ├───Mode: 1920x1080@60.00 preferred driver phsync pvsync 
+│   │   │   └───Gamma size: 256
+|   |   ...
+└───Planes
+    ├───Plane 0
+    │   ├───Object ID: 40
+    │   ├───CRTCs: {3}
+    │   ├───Legacy info
+    │   │   ├───FB ID: 0
+    |   ...
+```
+You want to make sure that you have:
+- 1 DRM Node (?) TOOD
```

Comment:
At least one.

(What's the TOOD? :wink:)

Reply:
OH! I meant to ask you here. it worked! 

What do you mean by "1 DRM Node" in the output? I don't see any listed there

Reply:
We meant `Node` here as the "drm node"